databroker/broker.py

```diff
@@ -0,0 +1,448 @@
+from __future__ import print_function
+import warnings
+import six  # noqa
+from collections import deque, defaultdict
+
+import tzlocal
+import logging
+import numbers
+from .core import (Header, _inspect_descriptor,
+                   get_events as _get_events,
+                   get_table as _get_table,
+                   restream as _restream,
+                   process as _process, Images)
+
+
+# Toolz and CyToolz have identical APIs -- same test suite, docstrings.
+try:
+    from cytoolz.dicttoolz import merge
+except ImportError:
+    from toolz.dicttoolz import merge
+
+
+try:
+    from functools import singledispatch
+except ImportError:
+    try:
+        # We are running on Python 2.6, 2.7, or 3.3
```

Comment:
Bluesky is a huge dependency that changes quite often under the hood. Clearly there is value in generating these documents via bs, however I don't think it's a good idea to have co-dependent projects. One can use mds without bs but not vice versa.